### PR TITLE
Drop old versions of php/sf

### PR DESCRIPTION
**Pull request by core23**
I am targeting this branch, because this is BC.

## Changelog

```markdown
### Removed
- Support for old versions of PHP and Symfony.
```

### CODE REVIEW

**Comment by core23** [comment]
The old PR had some sync problems https://github.com/sonata-project/SonataDoctrinePhpcrAdminBundle/pull/499#event-1293366039

File: composer.json [hunk 1]
```diff
@@ -16,15 +16,15 @@
         }
     ],
     "require": {
-        "php": "^5.6 || ^7.0",
+        "php": "^7.1",
```
Comment (ElectricMaxxx) [inline composer.json]: Not sure if we can drop that on next minor release.

File: composer.json [hunk 2]
```diff
@@ -16,15 +16,15 @@
...
         "doctrine/phpcr-bundle": "^1.1",
         "doctrine/phpcr-odm": "^1.3 || ^2.0",
         "sonata-project/admin-bundle": "^3.21.0",
         "sonata-project/block-bundle": "^3.0",
         "symfony-cmf/resource-rest-bundle": "^1.0",
         "symfony-cmf/tree-browser-bundle": "^2.0",
-        "symfony/framework-bundle": "^2.8 || ^3.0",
-        "symfony/property-access": "^2.8 || ^3.0"
+        "symfony/framework-bundle": "^2.8 || ^3.2",
```
Comment (ElectricMaxxx) [inline composer.json]: Why keeping 2.8 then, LTS?

Comment (greg0ire) [reply composer.json]: Yes, LTS

File: composer.json (hunk 1 above)
Comment (greg0ire) [reply composer.json]: We can, and did on all other packages.

**Comment by greg0ire** [comment]
@ElectricMaxxx please review again

**Comment by greg0ire** [comment]
Thanks @core23 !

**Pull request merged**